src/index.html

```diff
@@ -1,19 +1,52 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
+  <meta name="viewport" content="width=device-width,
           user-scalable=no,
           initial-scale=1.0,
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
 
-<script type="text/javascript" src="./main.js"></script>
+  <div class="imac-card" data-qa="card">
```

Comment:
`product-card` will be better.
In real project we don't know what it is gonna be, gadgets or clothes.